tests/conftest.py

```diff
@@ -111,11 +119,11 @@ def sample_custom_query_5():
 
 
 @pytest.fixture
-def sample_custom_query_4():
+def sample_custom_query_6():
```

Comment:
Probably something for later, but we could consider renaming the queries according to their functions

Reply:
Thanks the suggestion. Yeah, maybe addressable at a later time.